### DISCUSSION TITLE
Max FPS for V3

### DISCUSSION THREAD

**Discussion by endertable** [gtxaspec/wz_mini_hacks#113] [General]
Everywhere I read about the T31, it states it’s capable of up to 30 FPS. Is this true of the same hardware in the Wyze cam? If so has anyone found a way to override the 20 FPS that Wyze has it locked at?

**Comment by gtxaspec** [comment]
yes, but it requires heavy modification of the application and libraries. for the time being, not possible.  stay tuned.

**Comment by gtxaspec** [comment]
looking at the source code for the sensor modules, the gc2053 in the v3 is capable of 30 fps, and the jxf23 v2 should also be capable of 30, but the source code indicates its limited to 25.

I also remember hearing some jxf22 are included with the v2.  I'll keep looking into this.

**Reply by endertable** [reply]
This is exciting, is it just the V2 maxes at 25FPS or the V3 as well.  I am guessing we may have to wait for a newer Linux version to make this possible or is it more because of Wyze's binaries lockdown.

**Reply by Bob-AOL** [reply]
The V2 maxes at 15fps day and 10fps night stock. The V3 maxes at 20fps day and 15 night.

**Reply by baudneo** [reply]
> The V2 maxes at 15fps day and 10fps night stock. The V3 maxes at 20fps day and 15 night.

That's software limited though from what the others have commented. It sound like the hardware is capable of more. Also, you can currently turn off the fps drop at night in the .conf

**Comment by Bob-AOL** [comment]
Will need to check the conf for the night fps adjustment. 'Preciate the tip. I don't recall it being obvious -- but I completely overlooked the plainly detailed RTSP url info right under my nose, too. (Thanks again for your help there.) 

I did mention 'stock' regarding the fps limits. Have read the comments indicating the 30fps capability -- with a monster amount of work to make it happen as also indicated. That said, Blue Iris does report 22fps on occasion for my couple of v3s, but they usually hover around 19~20 in the 120kB/s neighborhood -- which I'm sure many folks would love to see on a regular basis. 

I'm curious if the 15fps on the v2s would be more easily increased than the 3s? That would be very cool.

**Comment by foureight84** [comment]
I've noticed that when I set the high-res RTSP feed to 30fps, the low-res feed seems to also get locked to 30fps as well. Although this is just from looking at the VLC stats for the feed. I did notice that if I were to set the low-res feed to a lower value like 5 or 10fps, I start to see a lot more dropped frames.
